### ISSUE TITLE
Cannot create a schedule using creation popup

### ISSUE THREAD

**Issue by lkmadushan** [nhnent/tui.calendar#122]
version: 1.2.2
browser: chrome

reproduction link: https://codesandbox.io/s/9j4yn57nnr

i'm getting below error when updating a schedule using creation popup.

```
Uncaught TypeError: Cannot read property 'id' of undefined
at ScheduleCreationPopup._onClickSaveSchedule
```

**Comment by dongsik-yoo** [comment]
@lkmadushan This is patched at v1.2.3. Please check it.

**Comment by lkmadushan** [comment]
@dongsik-yoo it's working. thanks!
